scripts/main.js

```diff
@@ -1091,6 +1098,36 @@
         return message;
     }
 
+    function getHuntDetails(message, response, journal) {
+        if (!message) {
+            return "";
+        }
+        switch (response.user.location) {
+            case "Bristle Woods Rift":
+                message = getBristleWoodsRiftHuntDetails(message, response, journal);
+                break;
+        }
+
+        return message;
+    }
+
+    function getBristleWoodsRiftHuntDetails(message, response, journal) {
+        message.hunt_details = {}
+        var quest = response.user.quests.QuestRiftBristleWoods
+        for (var key in quest.status_effects) {
+            if (!quest.status_effects.hasOwnProperty(key)) continue
+            message.hunt_details['effect_'+key] = quest.status_effects[key] === 'active'
+        }
+        message.hunt_details.has_hourglass = quest.items.rift_hourglass_stat_item.quantity >= 1
+        message.hunt_details.chamber_status = quest.chamber_status
+        if (quest.chamber_name === 'Acolyte') {
+            message.hunt_details.obelisk_charged = quest.obelisk_percent === 100
```

Comment:
I don't think this will work properly as the server response returns what is current, not what was during the hunt. So if the server returns 100, it's what it is after the hunt, not necessarily what was during the hunt. Not a stopper, just fyi.

Reply:
Correct, but in this case should make up for small offset in population distribution.